codecs/json-codec/src/test/java/software/amazon/smithy/java/json/JsonDocumentTest.java

```diff
@@ -423,40 +422,6 @@ public static List<Arguments> checkEqualitySource() {
                 Arguments.of("{\"foo\":\"foo\"}", "{\"foo\":\"bar\"}", false));
     }
 
-    @Test
-    public void onlyEqualIfBothUseTimestampFormat() {
```

Comment:
@mtdowling I think these tests are right. I don't think the equality of the documents should matter based on settings used to create them. As long as the deserialized contents are same, they should be same.

Reply:
Spoke privately. This is a grey area, disabling tests for now.